manual_flagging/flagging.py

```diff
@@ -22,10 +22,11 @@
 
 # Validate the input specification
 # Check housing_market_type
+# TODO: Add res_all and other res_type exclusivity check
```

Comment:
Ah, this TODO is an artifact I forgot to clean up. I believe since we have [this structure](https://github.com/ccao-data/model-sales-val/pull/104#discussion_r1523451996) we don't need to worry about having all 3 res configs at the same time.

Comment from the yaml: 
``` 
# "housing_market_type" and "run_tri" configurations determine the
# specificity of flagging. First, we retrieve the run tris, then run
# the flags on those tris for each selected market type, but only if
# the market type is present in the "stat groups" config under the
# "tri$n" value.
````

Reply:
I have a quick question about this, but I'll ask IRL!